### ISSUE TITLE
Cannot recognize file or directory

### ISSUE THREAD

**Issue by bradroush** [38leinaD/tango#8]
device: iPad2: iOS 5.0.1
desktop: Macbook Pro: Mac OS X 10.7.2

I tried to download and run your sample app on iOS 5 and it wouldn't display properly.  However, I put your library and obj c wrapper classes into a test project and I kept getting this error and output:

2011-12-11 14:10:48.888 TEST_Tango[4243:707] pressed
_tango_send(): Bytes to send (including nbt-header!): 51
_tango_send(): Sent 51 bytes.
_tango_send_and_receive(): Timeout during recv(): No such file or directory

I do have file sharing enabled on my desktop.  For the share input field I have tried:
\[host name].local[share folder]
\[host name].home[share folder]
\[host name].local\
\[host name].home\

**Comment by 38leinaD** [comment]
Try
\hostname\folder
as the common windows network share notation.
Though, i am not sure it will work with Apple's CIFS implementation coming with Mac OS X 10.7

Cheers,
Daniel

**Comment by bradroush** [comment]
Yeah, that didn't work either, same error and message.  Any suggestions?  Something I can do with the library?
